### ISSUE TITLE
Add a scrollbar, scrolling with C-d, C-u

### ISSUE THREAD

**Issue by MuntasirSZN** [soulis-1256/eagle.nvim#28]
Title ;) though scrolling should be implemented for keyboard only.

**Comment by MuntasirSZN** [comment]
Actually, when eagle win is open, scroll will keybinds, which i can't do. And scrollbar should be some like cmp.

**Comment by soulis-1256** [comment]
> Actually, when eagle win is open, scroll will keybinds, which i can't do. And scrollbar should be some like cmp.

You can definitely scroll within eagle, you just have to invoke it twice. One time to open it, and one more to focus inside of it. Then you can use whatever keybind you want to navigate within Neovim's normal or visual mode. As for the "scrollbar should be some like cmp" reauest, I can't say I understand what you mean. Can you elaborate?

**Comment by MuntasirSZN** [comment]
> > Actually, when eagle win is open, scroll will keybinds, which i can't do. And scrollbar should be some like cmp.
> 
> You can definitely scroll within eagle, you just have to invoke it twice. One time to open it, and one more to focus inside of it. Then you can use whatever keybind you want to navigate within Neovim's normal or visual mode. As for the "scrollbar should be some like cmp" request, I can't say I understand what you mean. Can you elaborate?

In blink.cmp docs view,i can scroll with the kebinds specified. Just a scrollbar blink.cmp has.

**Comment by soulis-1256** [comment]
![Image](https://github.com/user-attachments/assets/920e5b88-e1ab-4e7b-b011-56929a26751a)

I don't see a scrollbar (limited `max_height` to 2), nor a way to scroll with a keybind. I need more information about your use case.

**Comment by MuntasirSZN** [comment]
https://github.com/user-attachments/assets/2d0daf99-26ab-442c-a5d7-f31b0b25e1d8

**Comment by soulis-1256** [comment]
Ok, I understand your request now. However, scrolling this way isn't necessary when having a simple window. It makes sense in the context of the documentation window of blink, because there are two windows active at once. I will not be implementing this, because, as I said, it's very easy to go navigate inside eagle by just focusing on it (hover over with the mouse, or toggle twice with the keyboard).
